### DISCUSSION TITLE
aal-ʿAqrab

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#2695] [Translations]
At Transifex we got a possible error report:

[Euklid](https://www.transifex.com/user/profile/Euklid/): Looking at the Arabic word 'العقرب', it should be al-'Aqrab instead of aal-'Aqrab, since 'al' transliterates the Arabic article 'ال', which are the first two letters in the Arabic word.

@kajaji could you please check the transliteration?

**Comment by gzotti** [comment]
This is definitely a typo. Fixed in https://github.com/Stellarium/stellarium/commit/9f0e233197cbee8f3ca12b57f43b96ecf5c7f4e7
